design-system/src/index.ts

```diff
@@ -5,7 +5,7 @@ export {
   themes,
 } from './design-tokens';
 export { ThemeProvider, useTheme, withThemeContext } from './theme-provider';
-export type { ThemeName, TUseThemeResult } from './theme-provider';
```

Comment:
Exporting types directly from the package entry point causes issues with the preconstruct dev symlinks.

That's why we use the `./export-types` module trick.

Reply:
Thanks! 🙇